### PR TITLE
docs: minor updates in plugins NB 1

### PR DESCRIPTION
**Pull request by xczuba**

### CODE REVIEW

**Review by PierreLeveau — APPROVED** [review]

File: recipes/plugins_development.ipynb [hunk 1]
```diff
@@ -6,11 +6,11 @@
    "source": [
```
Comment (theodu) [inline recipes/plugins_development.ipynb:6]: <p>"For instance, you can trigger a plugin when a labeler clicks on Submit&nbsp;with the&nbsp;on_submit&nbsp;handler" is split on two line</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kili-technology/kili-python-sdk/pull/876/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (xczuba) [reply recipes/plugins_development.ipynb:6]: @theodu I'm not sure what the problem is. Can you add some more info?

Comment (theodu) [reply recipes/plugins_development.ipynb:6]: Sorry, i didn't give enough detail. I have the impression that the sentence that I sent in the message above is split 2 lines ? But may be it is only on the NB review app

Comment (xczuba) [reply recipes/plugins_development.ipynb:6]: Issue only exists in the review app, when new and old code is shown side-by-side. Closing.

**Review by theodu — APPROVED** [review]

**Pull request merged**